### ISSUE TITLE
Too few arguments to function Aoe\Restler\System\RestlerBuilderAware::__construct(), 0 passed

### ISSUE THREAD

**Issue by infabo** [AOEpeople/TYPO3_Restler#71]
Starting with version 10.2.0 the following error occurs:

![image](https://user-images.githubusercontent.com/3999104/146015156-7189689d-0dfb-4d37-8ef2-d544b029e0f7.png)

Basically it is just this config:

```yaml
routeEnhancers:
    RestlerEnhancer:
        type: Restler
        default: '.json'
```

**Comment by infabo** [comment]
It is a DI error. Did not investigate much, but I think it is because constructor injection does not work when creating instances with GeneralUtility - unless it is a public container.

**Comment by infabo** [comment]
Hi, forgot to mention:

- flushed cache in maintenance module from backend
- deleted `var` folder where DI cache resides.

No help either.

**Comment by soda-2005** [comment]
I have fixed the bug with version 10.2.3 of the restler-Extension.

Currently, github can not execute the tests - so the new version is currently NOT available in the TER for download - just for information.
